vtex-sites/nextjs.store
# Feat: Replace `RadioGroup` FSUI

## What's the purpose of this pull request?

Uses `RadioGroup` from the faststore/ui.

## How does it work?

Everything should look as before. (Visually).

## How to test it?

We don't use `RadioGroup` in our starter, but we use `RadioOption`, so you should search for a product with SkuSelector and verify if it works.

You can test through the preview link.
